src/System.ServiceModel.Duplex/tests/System.ServiceModel.Duplex.Tests.builds

```diff
@@ -6,6 +6,10 @@
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcore50;netcoreapp1.1</TestTFMs>
```

Comment:
from the logs of plain build.cmd the tests are doing this by default (2.0/2.0), however I'll happily add this to the .builds files.